Dockerfile

```diff
@@ -65,6 +65,11 @@ RUN mkdir -p ${GOPATH}/src/github.com/TheThingsIndustries/protoc-gen-gogottn &&
     make deps && \
     go install -ldflags '-w -s' .
 
+ARG PROTOC_GEN_GOVALIDATORS_VERSION
+RUN go get -u github.com/mwitkow/go-proto-validators/protoc-gen-govalidators && \
```

Comment:
Let's lock the ref here and use e.g. https://github.com/mwitkow/go-proto-validators/archive/0950a79900071e9f3f5979b78078c599376422fd.tar.gz
So just do a `curl -sSL https://github.com/mwitkow/go-proto-validators/archive/${PROTOC_GEN_GOVALIDATORS_VERSION}.tar.gz | tar xz --strip 1 -C ${GOPATH}/src/github.com/mwitkow/go-proto-validators`
instead of `go get`.
You may need to install dependencies.(`go get` is fine there)